src/styles/_toolbar.scss

```diff
@@ -0,0 +1,67 @@
+.toolbar {
+    background-color: $navy;
+    border-radius: $border-radius;
+    display: flex;
+    flex-direction: row;
+    font-family: $font-jost;
+    height: 70px;
+    width: 825px;
```

Comment:
In general, it's good practice to avoid fixed pixel widths like this if you can help it. Something to consider.

Reply:
Ok will keep that in mind. In this instance what would you use for height/width?

Reply:
If you're using flexbox or css-grid, you can have it expand the full width of the container. That way you avoid what we call "magic numbers". In general, it's always preferred to avoid "magic numbers" when possible.

If you want to make sure it only gets to a certain width, you could implement `max-width` for example.